src/dbcp/transform/justice40.py

```diff
@@ -151,10 +152,11 @@ def transform(raw_j40: dict[str, pd.DataFrame]) -> dict[str, pd.DataFrame]:
         "Percent of the Census tract that is within Tribal areas": "tract_within_tribal_areas_percent",
     }
 
-    out_df = raw_j40["justice40"].convert_dtypes()  # copy
+    out_df = raw_j40["justice40"].copy()
+    out_df = out_df.convert_dtypes()
     out_df.rename(columns=rename_dict, inplace=True)
     out_df.drop(columns="", inplace=True)
-    out_df.loc[:, "tract_id_fips"] = _fips_int_to_string(out_df.loc[:, "tract_id_fips"])
+    out_df["tract_id_fips"] = _fips_int_to_string(out_df.loc[:, "tract_id_fips"])
```

Comment:
I don't completely understand why but `.loc` wasn't actually assigning to the `tract_id_fips` column in `out_df`. Some sort of change in how writing to a copy works in the latest pandas.